samples/eShopLite/AppHost/Program.cs

```diff
@@ -32,6 +32,11 @@
 builder.AddProject<Projects.CatalogDb>("catalogdbapp")
        .WithReference(catalogDb);
 
-builder.AddProject<Projects.Aspire_Dashboard>("dashboard");
+// This project is only added in playground projects to support development/debugging
+// of the dashboard. It is not required in end developer code. Comment out this code
+// to test end developer dashboard launch experience. Refer to WorkloadAttributes.cs
+// for the path to the dashboard binary (defaults to a relative path to the artifacts
+// directory).
+//builder.AddProject<Projects.Aspire_Dashboard>("aspire-dashboard");
```

Comment:
```suggestion
builder.AddProject<Projects.Aspire_Dashboard>("aspire-dashboard");
```

We want this behavior locally. Especially in this project.

Reply:
Yeah ... I'll change this back once I've finished testing things out :)